ajax/libs/alertify.js/package.json

```diff
@@ -13,16 +13,30 @@
     "type": "git",
     "url": "https://github.com/fabien-d/alertify.js"
   },
-  "npmName": "alertify",
-  "npmFileMap": [
-    {
-      "basePath": "",
-      "files": [
-        "lib/*.js",
-        "themes/*.css"
-      ]
-    }
-  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/fabien-d/alertify.js.git",
+    "fileMap": [
+      {
+        "basePath": "lib",
+        "files": [
+          "alertify*"
+        ]
+      },
+      {
+        "basePath": "themes",
+        "files": [
+          "alertify*"
+        ]
```

Comment:
Let's make the pattern as 
```js
        "basePath": "",
        "files": [
          "themes/alertify*"
        ]
```
We don't let `themes` as the basePath.
Also, if you saw v0.4.0rc1 which get the files from `dist`, the file structure is 
```sh
├── 0.4.0rc1
│   ├── alertify.js
│   ├── alertify.min.js
│   └── themes
│       ├── alertify.bootstrap.css
│       ├── alertify.css
│       └── alertify.default.css
```

Thanks.